converters/fromZigbee.js

```diff
@@ -4703,6 +4703,29 @@ const converters = {
             }
         },
     },
+    legrand_power_alarm: {
+        cluster: 'haElectricalMeasurement',
+        type: ['attributeReport', 'readResponse'],
+        convert: (model, msg, publish, options, meta) => {
+            const payload = {};
+            
+            // 0xf000 = 61440
+            // This attribute returns usually 2 when power is over the defined threshold.
+            if (msg.data.hasOwnProperty('61440')) {
+                payload.power_alarm_active_value = msg.data['61440'];
+                payload.power_alarm_active = (payload.alarm_active_value > 0);
+            }
+            // 0xf001 = 61441
+            if (msg.data.hasOwnProperty('61441')) {
+                payload.power_alarm_enabled = msg.data['61441'];
+            }
+            // 0xf002 = 61442
+            if (msg.data.hasOwnProperty('61442')) {
+                payload.power_alarm_wh_threshold = msg.data['61442'];
```

Comment:
What does `wh` mean?

Reply:
Watts per hour.
1000 wh = 1 Kwh

Reply:
In the official app the jumps are not fine grain. The smaller is 1.5 Kwh, and sends to the device 1500 as value. Thus, I name as wh